src/views/ClusterViz/requests/InitProjectClusterRequest.js

```diff
@@ -0,0 +1,72 @@
+import { FgRestBuilder } from '#rs/utils/rest';
+import notify from '#notify';
+import schema from '#schema';
+
+import {
+    createUrlForInitClusterRequest,
+    createParamsForInitClusterRequest,
+} from '#rest';
+import _ts from '#ts';
+
+export default class InitProjectClusterRequest {
+    constructor(params) {
+        const {
+            stopRequestForClusterData,
+            startRequestForClusterData,
+            setState,
+        } = params;
+        this.stopRequestForClusterData = stopRequestForClusterData;
+        this.startRequestForClusterData = startRequestForClusterData;
+        this.setState = setState;
+    }
+
+    success = projectId => (response, code) => {
+        try {
+            schema.validate(response, 'initClusterRequest');
+            const { cluster_model_id: modelId } = response;
+            this.stopRequestForClusterData();
+            this.startRequestForClusterData(modelId, projectId);
+        } catch (err) {
+            console.warn(err);
+        }
+    }
+
+    failure = (response) => {
+        notify.send({
+            title: _ts('clusterViz', 'clusterVizTitle'),
+            type: notify.type.ERROR,
+            message: response.error,
```

Comment:
transform the failure response error using alterAndCombineResponseError function.